nautobot/utilities/templatetags/helpers.py

```diff
@@ -304,32 +304,27 @@ def percentage(x, y):
 
 @library.filter()
 @register.filter()
-def get_docs(model):
-    """Render and return documentation for the specified model.
+def get_docs_url(model):
+    """Return the documentation URL for the specified model.
+
+    Nautobot Core models have a path like docs/models/{app_label}/{model_name}
+    while plugins will have {app_label}/docs/models/{model_name}.
 
     Args:
         model (models.Model): Instance of a Django model
 
     Returns:
-        str: documentation for the specified model in Markdown format
+        str: Url for the documentation of the object.
 
     Example:
-        >>> get_docs(obj)
-        "some text"
+        >>> get_docs_url(obj)
+        "static/docs/models/dcim/site.html"
     """
-    path = "{}/models/{}/{}.md".format(settings.DOCS_ROOT, model._meta.app_label, model._meta.model_name)
-    try:
-        with open(path, encoding="utf-8") as docfile:
-            content = docfile.read()
-    except FileNotFoundError:
-        return "Unable to load documentation, file not found: {}".format(path)
-    except IOError:
-        return "Unable to load documentation, error reading file: {}".format(path)
-
-    # Render Markdown with the admonition extension
-    content = markdown(content, extensions=["admonition", "fenced_code", "tables"])
-
-    return mark_safe(content)
+    return (
+        f"{settings.STATIC_URL}{f'{model._meta.app_label}/' if model._meta.app_label in settings.PLUGINS else ''}"
+        f"docs/models/{f'{model._meta.app_label}/' if model._meta.app_label not in settings.PLUGINS else ''}"
+        f"{model._meta.model_name}.html"
+    )
```

Comment:
Is there a technical reason that plugin docs need to be in `app_label/docs/models/` while core docs get to be in `docs/models/app_label/`? There shouldn't be an `app_label` collision between core and plugin anyway (you can't really make a plugin called `dcim`) so can we establish a pattern here where they *don't* need to be handled differently?

Reply:
Yes, so while the models won't conflict, the docs will. For instance, a plugin will probably want docs/index.html, which would conflict with core. In addition, this is how plugins currently do the static directory(at least all I have seen).

Reply:
Good points, thanks!